packages/gamut/src/Form/FormGroupDescription.tsx

```diff
@@ -1,15 +1,19 @@
-import cx from 'classnames';
+import { system } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
 import React from 'react';
 
-import styles from './styles/FormGroupDescription.module.scss';
+import { formBaseStyles } from './styles/shared-system-props';
 
 export type FormGroupDescriptionProps = React.HTMLAttributes<HTMLDivElement> & {
   className?: string;
 };
 
+const StyledDiv = styled.div<FormGroupDescriptionProps>`
+  ${system.css(formBaseStyles)}
+`;
```

Comment:
Do we even need the extra component now?

```suggestion
export const FormGroupDescription = styled.div(system.css(formBaseStyles));
```

Reply:
🙏

Reply:
we may still need to support className but i'll see if it's actually used anywhere...